community/kernel/src/test/java/org/neo4j/kernel/RecoveryCorruptedTransactionLogIT.java

```diff
@@ -395,7 +395,31 @@ public void repetitiveRecoveryIfCorruptedLogsWithCheckpoints() throws IOExceptio
 
         while ( logFiles.getHighestLogVersion() > 0 )
         {
-            truncateBytesFromLastLogFile( 1 + random.nextInt( 100 ) );
+            int bytesToTrim = 1 + random.nextInt( 100 );
+            System.out.println("Trim: " + bytesToTrim);
```

Comment:
Is this sysout here on purpose? If yes, maybe we should add a rule to suppress output?